pkg/utils/credentials.go

```diff
@@ -26,6 +26,7 @@ func ReplaceCarriageReturn(data map[string][]byte) map[string][]byte {
 
 	return data
 }
+//TRIGGER CI
```

Comment:
What's this for?

Reply:
I'll remove that once we see things working correctly.  The e2e jobs were not triggered w/ the update to the .sh file :/